wagtail_localize/tests/test_locale_model.py

```diff
@@ -0,0 +1,74 @@
+from django.conf import settings
+from django.test import TestCase, override_settings
+from django.utils import translation
+from wagtail.core.models import Page
+
+from wagtail_localize.models import Locale
+from wagtail_localize.test.models import TestPage, TestHomePage
+
+
+def make_test_page(**kwargs):
+    root_page = Page.objects.get(id=1)
+    kwargs.setdefault("title", "Test page")
+    return root_page.add_child(instance=TestPage(**kwargs))
+
+
+class TestLocaleModel(TestCase):
+    def setUp(self):
+        language_codes = dict(settings.LANGUAGES).keys()
+
+        for language_code in language_codes:
+            Locale.objects.update_or_create(
+                language_code=language_code, defaults={"is_active": True}
+            )
+
+    def test_default(self):
+        locale = Locale.objects.default()
+        self.assertEqual(locale.language_code, "en")
+
+    @override_settings(LANGUAGE_CODE="fr-ca")
+    def test_default_doesnt_have_to_be_english(self):
+        locale = Locale.objects.default()
+        self.assertEqual(locale.language_code, "fr")
+
+    def test_get_active_default(self):
+        self.assertEqual(Locale.get_active().language_code, "en")
+
+    def test_get_active_overridden(self):
+        with translation.override("fr"):
+            self.assertEqual(Locale.get_active().language_code, "fr")
+
+    def test_get_display_name(self):
+        locale = Locale.objects.get(language_code="en")
+        self.assertEqual(locale.get_display_name(), "English")
+
+    def test_get_display_name_for_unconfigured_langauge(self):
+        # This language is not in LANGUAGES so it should just return the language code
+        locale = Locale.objects.create(language_code="foo")
+        self.assertIsNone(locale.get_display_name())
```

Comment:
The test comment mentions returning just the language code, but it returns `None`?

Reply:
Ahh, `__str__` does that, but this should return `None`